src/status_im/chat/views/input/input_actions.cljs

```diff
@@ -4,10 +4,10 @@
             [reagent.core :as r]
             [re-frame.core :refer [subscribe dispatch]]
             [taoensso.timbre :as log]
-            [status-im.ui.components.react :refer [view
+            [status-im.ui.components.react :refer [view]
```

Comment:
ops

Reply:
Also alias instead of refer, if its refactoring PR it's important to clean up